File: .claude/plugins/openai-agent-builder/references/project-templates/python-multi-agent/triage.py
"""{{PROJECT_NAME}} — Triage agent with handoffs to specialists."""
from agents import Agent
from agents.extensions.handoff_prompt import prompt_with_handoff_instructions
from specialists.specialist_1 import specialist_1
from specialists.specialist_2 import specialist_2

specialists = [specialist_1, specialist_2]

triage_agent = Agent(
    name="{{AGENT_NAME}}",
    instructions=prompt_with_handoff_instructions(
        "You are a triage agent. Analyze the user's request and route them to the most appropriate specialist. "
        "If the request doesn't match any specialist, answer it yourself.",
        specialists,
    ),
    handoffs=specialists,
    model="gpt-4.1",
)

File: .claude/plugins/openai-agent-builder/references/project-templates/typescript-multi-agent/src/triage.ts
import { Agent, handoff } from '@openai/agents';
import { specialist1 } from './specialists/specialist1';
import { specialist2 } from './specialists/specialist2';

export const triageAgent = new Agent({
  name: '{{AGENT_NAME}}',
  instructions:
    'You are a triage agent. Analyze the user\'s request and route them to the most appropriate specialist. If the request doesn\'t match any specialist, answer it yourself.',
  handoffs: [
    handoff(specialist1, { description: '{{SPECIALIST_1_HANDOFF_DESCRIPTION}}' }),
    handoff(specialist2, { description: '{{SPECIALIST_2_HANDOFF_DESCRIPTION}}' }),
  ],
  model: 'gpt-4.1',
});
